### PR TITLE
ci: update signaling

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

contra-part for https://github.com/gridai/base-images/pull/66

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda

### CODE REVIEW

**Comment by Borda** [comment]
checking the pass:
image

**Review by awaelchli — APPROVED** [review]

File: .github/workflows/release-pypi.yml [hunk 1]
```diff
@@ -115,18 +111,21 @@ jobs:
...
+        git checkout -b ${BRANCH_NAME}
         git add versions.json
-        git commit -m "bumping lightning version -> ${TAG}"
-        cat versions.json
-    - name: GIT Push
-      run: |
         git status
-        git push
+        git commit -m "bumping lightning version -> ${TAG}"
+        git push -u origin ${BRANCH_NAME} -f
```
Comment (akihironitta) [inline .github/workflows/release-pypi.yml:125]: Does the user pushing the commit have the write permission (probably admin-level is required to push to the main branch if the main branch is protected)?

**Review by akihironitta — APPROVED** [review]
LGTM!

File: .github/workflows/release-pypi.yml (hunk 1 above)
Comment (akihironitta) [reply .github/workflows/release-pypi.yml:125]: Oh it's changed to `trigger/lightning-` :) LGTM!

**Pull request merged**